smart_campus/app/views.py

```diff
@@ -668,6 +660,45 @@ def reward_add_page(request):
 
 
 @login_required
+def reward_edit_page(request, pk):
+    reward = get_object_or_404(Reward, pk=pk)
+    if request.method == 'POST':
+        reward_form = RewardForm(request.POST, request.FILES, instance=reward)
+        print(request.POST)
```

Comment:
Why is this `print` statement needed?

Reply:
Just the debug part didn't find out before commit, my fault.